website/docs/telemetry.md

```diff
@@ -35,23 +35,30 @@ Argument values, data field names, successful responses, or error messages are n
 ## Version notification
 
 When a FerretDB update is available, the telemetry service sends periodic reports containing information about the latest FerretDB version.
-
-This information is logged in the server logs, `startupWarnings`, and `serverStatus` command output.
+This information is logged in the server logs and `startupWarnings` command output.
 
 While you may not upgrade to the latest release immediately,
 ensure that you update early to take advantage of recent bug fixes, new features, and performance improvements.
 
 ## Configure telemetry
 
 The telemetry service has three state settings: `enabled`, `disabled`, and `undecided` (default).
+The latter acts as if the telemetry reporter is `enabled` with two differences:
```

Comment:
What "the latter" refers to? If it's about `undecided`, let's say `undecided` for clarity.

Reply:
I'm not sure if that would improve clarity (what else "the latter" could mean?), but that would definitely increase tautology by using the same word next to each other